.travis.yml

```diff
@@ -0,0 +1,26 @@
+sudo: required
+dist: trusty
+
+
+services:
+  - docker
+
+language: cpp
+
+env:
+  matrix:
+    - COMPILER=gcc
+
+before_install:
+  - wget https://gitlab.cern.ch/lhcb-core/LbDocker/raw/master/scripts/lb-docker-run 
```

Comment:
Seems very overengineered just to get a few options passed to the `docker run` command...